.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,38 @@
+name: Daily build
```

Comment:
Do we need daily build workflow now?

Reply:
Added the daily build because' stdlib-publish-snapsho't build has only the Ubuntu build, just wanted to check the Windows build is also successful at least twice a day.

Reply:
Ack